include/revng/ADT/RecursiveCoroutine.h

```diff
@@ -13,3 +15,12 @@
 #include "RecursiveCoroutine-coroutine.h"
 
 #endif
+
+// This could be dropped right now, as RecursiveCoroutines can be called just
+// like regular functions. However, I'll leave it because rc_run is a nice
+// marker to see in code to realize that a call is actually a call to a
+// RecursiveCoroutine.
```

Comment:
`:OOOOOOOOOOO`

I'm honestly in favor of dropping it at this point. One macro less.